scipy/stats/tests/test_stats.py

```diff
@@ -2084,7 +2084,10 @@ def test_scale(self):
                                     np.array([2, np.nan, 2]) / 1.3489795)
                 assert_equal(stats.iqr(x, axis=1, scale=2.0, nan_policy='propagate'),
                              [1, np.nan, 1])
-                _check_warnings(w, RuntimeWarning, 6)
+                if numpy_version <= '1.16.6':
```

Comment:
Probably good to check this addition of mine---apparently less warnings  are expected here for newer NumPy versions. I do see that the test code above is also littered with NumPy version checks, so perhaps not surprising.